packages/webviz-core/src/panels/ThreeDimensionalViz/commands/LaserScans.js

```diff
@@ -19,6 +19,19 @@ import {
 import { getGlobalHooks } from "webviz-core/src/loadWebviz";
 import type { LaserScan } from "webviz-core/src/types/Messages";
 
+const colorRGBAStructToArray = (struct) => {
+  return [struct.r, struct.g, struct.b, struct.a];
+};
```

Comment:
Could use [toRGBA](https://github.com/cruise-automation/webviz/blob/master/packages/regl-worldview/src/utils/commandUtils.js#L51-L53) from regl-worldview: `import {toRGBA} from 'regl-worldview';`

Reply:
Ah, thanks, I briefly looked for something like that, but I couldn't find it.